tests/controller_tests/test_download.py

```diff
@@ -34,3 +44,144 @@ def test_download_excel_format_empty_db(test_client):
     response = test_client.get("/download?file_format=xlsx")
     assert response.status_code == 200
     assert response.content_type == EXCEL_MIMETYPE
+
+
+def test_download_contains_data_from_the_correct_time_period(seeded_test_client):
```

Comment:
this might be nit-picking, but this test doesn't appear to actually test what's it's name suggests? Ie it only actually asserts on stuff NOT in the download?